appconfiguration/spring-cloud-azure-feature-management-web/spring-cloud-azure-targeting-filter-web-sample/src/main/java/com/example/demo/configuration/AppConfiguration.java

```diff
@@ -0,0 +1,17 @@
+package com.example.demo.configuration;
+import com.azure.spring.cloud.feature.management.filters.TargetingFilter;
+import com.azure.spring.cloud.feature.management.targeting.TargetingEvaluationOptions;
+import com.example.demo.impl.TargetingContextImpl;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+
+@Configuration
+public class AppConfiguration {
+
+    @Bean(name = "targetingFilter")
+    @Scope("request")
```

Comment:
NIT: You don't need the name value if the field matches method name.
NIT: You can use `@RequestScope` instead.

Reply:
Updated